### PR TITLE
367 Chambers Dropdown

### PR DESCRIPTION
**Pull request by wbyoko**
Screenshot 2019-03-28 19 13 44
Screenshot 2019-03-28 19 13 37
Screenshot 2019-03-28 19 13 32
Screenshot 2019-03-28 19 13 19
Screenshot 2019-03-28 19 13 13
Screenshot 2019-03-28 19 13 07

### CODE REVIEW

File: web-client/src/presenter/sequences/updateForwardFormValueSequence.js [hunk 1]
```diff
@@ -1,6 +1,34 @@
+/* eslint-disable sort-keys-fix/sort-keys-fix */
```
Comment (rachaelparris) [inline web-client/src/presenter/sequences/updateForwardFormValueSequence.js:1]: Is this here on purpose? Or just forgot to remove it?

File: web-client/src/presenter/sequences/updateMessageValueSequence.js [hunk 2]
```diff
@@ -0,0 +1,34 @@
+/* eslint-disable sort-keys-fix/sort-keys-fix */
```
Comment (rachaelparris) [inline web-client/src/presenter/sequences/updateMessageValueSequence.js:1]: Same as other comment. :)

File: web-client/src/views/DocumentDetail/PendingMessages.jsx [hunk 3]
```diff
@@ -300,15 +301,67 @@ class PendingMessagesComponent extends React.Component {
...
+                        {!workItemMetadata.showChambersSelect && (
+                          <div className="usa-input-error-message beneath">
+                            {validationErrors[workItem.workItemId] &&
+                              validationErrors[workItem.workItemId].section}
+                          </div>
+                        )}
+                      </div>
+
+                      {workItemMetadata.showChambersSelect && (
+                        <div
```
Comment (rachaelparris) [inline web-client/src/views/DocumentDetail/PendingMessages.jsx:317]: I'm just curious - is it ok to have this `showChambersSelect` stored on the workItem in the state? Could this end up in the actual data in dynamo somehow? Or does that even matter?

Comment (wbyoko) [reply web-client/src/views/DocumentDetail/PendingMessages.jsx:317]: This is workItemMetadata. It's being used as a place to put ui state in cerebral.

File: web-client/src/presenter/sequences/updateForwardFormValueSequence.js (hunk 1 above)
Comment (wbyoko) [reply web-client/src/presenter/sequences/updateForwardFormValueSequence.js:1]: The code becomes incongruent when the keys are in alpha order.  

as in...
the `default` case should be last in the path list. 
the `yes` case should come before the `no` case.

File: web-client/src/views/DocumentDetail/PendingMessages.jsx (hunk 3 above)
Comment (rachaelparris) [reply web-client/src/views/DocumentDetail/PendingMessages.jsx:317]: Ok, ignore me. 😛

**Review by rachaelparris — APPROVED** [review]

**Review by codyseibert — APPROVED** [review]

**Pull request merged**